specification/delegates/RegisterDelegate/RegisterDelegateTransactionDto.yaml

```diff
@@ -0,0 +1,15 @@
+allOf:
+  - $ref: "../../common/transactions/QueuedTransaction.yaml"
+  - type: object
+    required:
+      - asset
+      - type
+    properties:
+      asset:
+        $ref: "../../common/transactions/assets/RegisterDelegateAsset.yaml"
+      type: # overrides QueuedTransaction.yaml
+        type: integer
+        minimum: 2
+        maximum: 2
+        example: 2
+        description: Always equal to `2`
```

Comment:
Does this endpoint respond return `success` and `nodeTimestamp`? Then describe them in properties

Reply:
These properties are described inside `RegisterDelegateResponseDto.yaml`